active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,305 @@
+
+### Keyboarding
+* Pager Control should be added as a navigation landmark. 
+* End users should be able to tab to put focus on the first item in the control. 
+* Left and right arrows will navigate between the items in the control. 
+* The default accessibility behavior of the combo box and number box will be applied to this control.  
+
+### Narrator
+* When the focus is on the control, narrator will announce "pager". 
+* When the focus is on the first, last, previous, or next button narrator will announce "first page", "last page", "next page", or "previous page". 
+        * If the buttons have text properties set by the developer, narrator will announce that text instead of the default announcement. 
+* When the focus is on a number button, narrator will announce "page x of y". 
+* Combo box and number box modes will use the default narrator announcements for combo box and number box when the user puts focus on them. 
+* When the user enters a value in to the number box, narrator will announce the number entered. 
+
+### GamePad 
+* Pager Control will not have engagement. The internal components (ComboBox and NumberBox) will keep their current engagement and accessibility features for game pad. 
+* The user can use the right and left arrow keys to navigate between the buttons and use the A button to select an item. 
+* Guidance will be added for app developers that are targeting Xbox applications to not place items to the left or right of the pager control. 
+        * Text added to guidance: “If you are developing an application that will target Xbox, we recommend not placing any UI elements to the left or right of the pager control so the end user does not have to navigate through the pager control in order to reach the next UI element.” 
+
+### Localization
+* The numbers will follow standard behavior when localized to/from a language that uses a numeral system that differs from Western Arabic. 
+* The prefix and suffix text will follow standard mirroring behavior when localized to/from languages that are RTL or LTR. 
+
+## Data and Intellegence Metrics 
+
+Adoption and validation of the control in the community
+* Measurement: Use telemetry to determine how many first and third party developers are using the control in preview. 
+* Measurement: Count of posts providing feedback to us through Github and Discord. 
+
+Questions that can be answered through telemetry metrics to help focus future features and improvements: 
+Which display mode is the most commonly used? 
+* Measurement: Count of how many applications include the pager control type broken out by display mode.
+
+How often are developers using this control with an indefinite number of pages?
+* Measurement: Count of how many applications include the pager control type and have the NumberOfPages property empty.
+
+
+## Open Questions 
+* How does narrator handle localization? In the “page x of y” scenario how would that work if we switch from LTR to RTL?
```

Comment:
Does RTL or LTR make a difference in this scenario? Isn't that something that we are providing through the AutomationPeer which provides the PositionInSet and SizeOfSet values to the narrator?

Reply:
I don't think anything special is needed here for RTL, but @MikeHillberg can say for sure. If the items get flipped, the elements would still have the correct position/size properties still.

Reply:
Yes but I am not sure if narrator will correctly read page x of y in that case if the localized language is RTL. @StephenLPeters mentioned this before and I wanted to add it as an open question to make sure I talk with accessibility about it.

Reply:
@chingucoding you are right this is not an issue since we are using the PositionInSet property. Good catch!